src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/acl_editor.cljs

```diff
@@ -52,22 +36,25 @@
              {:ref (str "acl-key" i)
               :predicates [(input/valid-email-or-empty "User or Group ID")]
               :style {:float "left" :width column-width :color "black"
-                      :backgroundColor (when (< i (:count-orig @state)) (:background-gray style/colors))}
-              :disabled (< i (:count-orig @state))
+                      :backgroundColor (when (:read-only? acl-entry)
+                                         (:background-gray style/colors))}
+              :disabled (:read-only? acl-entry)
               :spellCheck false
-              :defaultValue (:email acl-entry)}]
+              :value (:email acl-entry)
+              :onChange #(swap! state assoc-in [:acl-vec i :email] %)}]
             (style/create-select
               {:ref (str "acl-value" i)
                :style {:float "right" :width column-width :height 33}
-               :defaultValue (:accessLevel acl-entry)}
-              access-levels)
+               :value (:accessLevel acl-entry)
+               :onChange #(swap! state assoc-in [:acl-vec i :accessLevel]
+                                 (.. % -target -value))}
+              (reduce (fn [r x] (assoc r x x)) {} access-levels))
```

Comment:
ah - this could just use create-identity-select instead of transforming access-levels into a x->x map. I'm fine with either approach - just an example of how create-select and create-identity-select should be combined.